include/sound/soc-acpi.h

```diff
@@ -193,6 +194,15 @@ struct snd_soc_acpi_link_adr {
  *  is not constant since this field may be updated at run-time
  * @sof_tplg_filename: Sound Open Firmware topology file name, if enabled
  * @tplg_quirk_mask: quirks to select different topology files dynamically
+ * @get_sub_tplg_files: This is an optional callback, if specified then instead of
```

Comment:
Yeah, but `sub` to what? There is a main tplg file which these are subs to?
`get_function_tplg_files` would describe what this does better.
For me the tplg sounds sloppy compared to topology, but if the concern is length, then it is fine.

Reply:
Renamed to `get_function_tplg_files` now